### PR TITLE
Adding and re-organizing lattice protocol documentation.

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

**Comment by autodidaddict** [comment]
ping @brooksmtownsend

File: content/reference/lattice-protocols/rpc.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "RPC"
+date: 2018-12-29T11:02:05+06:00
+weight: 2
+draft: false
+---
+
+At its core, the _lattice_ is essentially an RPC bus layer built on top of the NATS message broker. When the lattice
+is enabled, participants in lattice RPC must conform to a set of standards in order to be able to send and handle messages.
```
Comment (brooksmtownsend) [inline content/reference/lattice-protocols/rpc.md:9]: For clarity, perhaps either the title or first mention of RPC should be `Remote Procedure Call (RPC)`

File: content/reference/lattice-protocols/control-interface.md [hunk 2]
```diff
@@ -13,15 +13,14 @@ The following is a list of the operations supported by the control interface.
 
 ### NATS Control Interface
 
-All of the control interface messages published on NATS topics use a standard prefix. This prefix is `wasmbus.ctl.{namespace}` where `namespace` is a string used to differentiate one lattice from another. Note that this namespace must correspond to the namespace prefix of the lattice you intend to control. As an example, if your lattice namespace prefix is `prod`, then you would subscribe to the lattice event stream at `wasmbus.ctl.prod.events`.
+All of the control interface messages published on NATS topics use a standard prefix. This prefix is `wasmbus.ctl.{namespace}` where `namespace` is a string used to differentiate one lattice from another. Note that this namespace must correspond to the namespace prefix of the lattice you intend to control.
 
 ⚠️ You need to ensure that your namespace prefix does not contain any characters that cannot appear in a NATS subject.
 
-All control interface messages are serialized as _message pack_ payloads that conform to the schema found in the [actor interfaces](https://github.com/wasmcloud/actor-interfaces) repository.
```
Comment (brooksmtownsend) [inline content/reference/lattice-protocols/control-interface.md:20]: This is a comment for the above line:
```
⚠️ You need to ensure that your namespace prefix does not contain any characters that cannot appear in a NATS subject.
```
We don't want to be in the business of keeping a full list of not-allowed things, but could we say something like "Combinations of ASCII letters and numbers are an example of valid characters for NATS subjects" or point out any common mistakes (no trailing spaces, etc)

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
two quick requests

**Review by brooksmtownsend — APPROVED** [review]
I like it

**Pull request merged**